code/processes/wdb.q

```diff
@@ -477,7 +479,10 @@ replayupd:{[f;t;d]
 	/- if the data count is greater than the threshold, then flush data to disk
 	if[(rpc:count[value t]) > lmt:maxrows[t];
 		.lg.o[`replayupd;"row limit (",string[lmt],") exceeded for ",string[t],". Table count is : ",string[rpc],". Flushing table to disk..."];
-		savetables[savedir;getpartition[];0b;t]]	
+		/- if datastriping is on then filter before savedown to the tailDB, if not save down to wdbhdb
+    $[.ds.datastripe;
+     (.ds.savetablesoverperiod[.ds.td;;.z.p+10:00;.z.p]each .wdb.tablelist[];.ds.applyfilters[ enlist t;.sub.filterdict]);
+     savetables[savedir;getpartition[];0b;t]]]	
```

Comment:
indentation inconsistent

Reply:
What is the convention in this case? Should the 2 lines following the conditional both be indented?

Reply:
My main concern here is that these three lines uses spaces while the surrounding all use tabs

But yes imo the branches of the conditional should be indented further than the initial line